.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: c
+dist: bionic
+compiler: gcc
+before_install:
+  - src=$(pwd)
+  - mkdir /tmp/linux
```

Comment:
It would bee better to use `/tmp/linux` from a variable.

Reply:
Sure. I'll also fix the other variable names.